GSA/digitalgov.gov
# Remove hidden section

## Summary

Removed hidden section. Closes #7645

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-hidden-code-from-terms-page/topics)

 
### Solution
Removed hidden section. Closes #7645

### How To Test

1. Go to the [topics page](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/rs-remove-hidden-code-from-terms-page/topics)
2. Notice nothing is changed.
3. [Click Here To View The Commit](https://github.com/GSA/digitalgov.gov/commit/ba4c03b61b71637a44845a3bb73094d6ae721682)
4. Notice dead code is removed
---

### Dev Checklist
